Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.8.5-alpine
+
+RUN apk update \
+    && apk add --virtual build-deps gcc python3-dev musl-dev \
+    && apk add postgresql \
+    && apk add postgresql-dev \
+    && pip install psycopg2 \
+    && apk add jpeg-dev zlib-dev libjpeg \
+    && pip install Pillow \
+    && apk del build-deps
+
+COPY ./requirements.txt requirements.txt
+COPY ./livestream_saver.py livestream_saver.py
+COPY ./livestream_saver livestream_saver
+RUN python3 -m pip install --upgrade pip
+RUN python3 -m pip install -r requirements.txt
+
+COPY ./livestream_saver.cfg.template ./config.yml
```

Comment:
Shouldn't this be the following instead?
```Dockerfile
COPY ./livestream_saver.cfg.template ./livestream_saver.cfg
```

Reply:
I'm sorry I have an examination at university. I found more bugs in my code. Please close this PR and next week I create a new one with more improvements and fixes.

But yes you are right.

Reply:
It's alright. You can push changes to that same branch and it will be updated. There is no rush.